JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt

```diff
@@ -113,14 +142,15 @@ private fun PublicationsTab() {
 private fun TabWithTopics(tabname: String, topics: List<String>) {
     VerticalScroller {
         Column {
-            Spacer(LayoutHeight(16.dp))
+            Spacer(modifier = LayoutHeight(16.dp))
```

Comment:
Couldn't this be `LayoutPadding` on the parent?

Reply:
Yea good catch, I tried refactoring this before reverting to original and didn't think of that